osu.Game/Overlays/Mods/ModSelectOverlayStatics.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Game.Configuration;
+
+namespace osu.Game.Overlays.Mods
+{
+    /// <summary>
+    /// Stores global mod overlay statics. These will not be stored after disposal of <see cref="ModSelectOverlay"/>
+    /// </summary>
+    public class ModSelectOverlayStatics : InMemoryConfigManager<Static>
```

Comment:
I'm... really not sure that we want to have another one of these. I'd use `SessionStatics` at the least, and maybe even consider renaming/retrofitting `LastHoverSoundPlaybackTime` to handle this too, to have just one of these around ever. A sample is a sample is a sample, I don't really see much point in hemming and hawing whether the sample being debounced is a hover or whatever else.

In fact, toasts do this already, and I see no reason not to do that here too.

https://github.com/ppy/osu/blob/659a04206544eb3c16b7d0433b4794734221275f/osu.Game/Overlays/OSD/TrackedSettingToast.cs#L96-L100

Reply:
I thought about moving it to `SessionStatics` but didn't because mod panel isn't really game-wide (it just not fitting there).

Also using `LastHoverSoundPlaybackTime` will allow users to silently activate panels:

https://github.com/ppy/osu/assets/50776304/ca6acc94-a389-46f8-ad94-9d3224f9aa7e

Reply:
Fair point on the reuse of `LastHoverSoundPlaybackTime`, but I guarantee you that the chance that nothing else will ever use `ModSelectOverlayStatics` is 99.9%. `SessionStatics` is a hodgepodge of everything anyways, claiming that something "doesn't fit" there is really strange.

Let's not multiply entities until it's strictly required.